### ISSUE TITLE
How to add this plugin on Nativescript-AngularJS

### ISSUE THREAD

**Issue by thelastsummer** [lochstar/nativescript-cast#1]
there are methods to add on nativescript-core and nativescript-vue, but no for nativescript-angularjs
is there any way to add it on  nativescript-angular.js?

**Comment by lochstar** [comment]
I haven't played around with Nativescript Angular yet. [Adding support looks pretty straight forward](https://docs.nativescript.org/plugins/angular-plugin). I'll try to make some time this week to get it done.

**Comment by thelastsummer** [comment]
when I installed plugin, I get this issue.
```
System.err: android.content.res.Resources$NotFoundException: Resource ID #0xfffffff6
System.err:     at android.content.res.ResourcesImpl.getValue(ResourcesImpl.java:195)
System.err:     at android.content.res.Resources.loadXmlResourceParser(Resources.java:2133)
System.err:     at android.content.res.Resources.getAnimation(Resources.java:1166)
System.err:     at android.view.animation.AnimationUtils.loadAnimation(AnimationUtils.java:107)
System.err:     at android.support.v4.app.FragmentManagerImpl.loadAnimation(FragmentManager.java:1073)
System.err:     at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1554)
System.err:     at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1595)
System.err:     at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:758)
System.err:     at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2363)
System.err:     at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2149)
System.err:     at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2103)
System.err:     at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2013)
System.err:     at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:710)
System.err:     at android.os.Handler.handleCallback(Handler.java:789)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:98)
System.err:     at android.os.Looper.loop(Looper.java:164)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:6541)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```

How can I fix it?

**Comment by lochstar** [comment]
Did you set your

**Comment by thelastsummer** [comment]
yes.
I think this is relative with android version and sdk.
when I tried to this plugin on old module(android version: 4.1.1), the issue has gone.

**Comment by jenyaivanova521** [comment]
I tried this plugin, and the chromecast button is there but it looks to be disabled?
Nothing happens when I click on it.

**Comment by lochstar** [comment]
@thelastsummer
Yep, I don't had it working with tns-core-modules 5+ yet.

@jenyaivanova521 
It will show as disabled if there are no Chromecast devices on the network.

**Comment by jenyaivanova521** [comment]
Just tested it again and still disabled. Chromecast works in Netflix and youtube

**Comment by lochstar** [comment]
Are you testing on a device or the emulator? The emulator doesn't work with Chromecast.

The enable/disable is handled by the framework, the plugin doesn't touch it.

**Comment by jenyaivanova521** [comment]
yeah.
it is enabled on device. but I got another issue
![Screenshot_20190314-145058_videoondemand (1)](https://user-images.githubusercontent.com/43125238/54373532-a3b36000-463a-11e9-8d7a-35639807ef53.jpg)

**Comment by lochstar** [comment]
`The activity must be a subclass of FragmentActivity`, this is mentioed in the Android usage instructions. Please check the README. If you're still having problems, please open a new issue as this one is in about Angular support.

**Comment by jenyaivanova521** [comment]
yes,
I copied App_Resources\Android folder on your demo project and replaced fully on my project
so that is working!

**Comment by lochstar** [comment]
Just published `0.1.1` with Angular support.
